migrations/sql/R__stored_procedure_for_permit_and_permitee_information.sql

```diff
@@ -593,11 +594,11 @@ CREATE OR REPLACE FUNCTION transfer_permit_permitee_information() RETURNS void A
             SELECT count(*) FROM permit_amendment INTO old_amendment_row;
 			
             RAISE NOTICE '.. Update ETL_PERMIT and all_permit_info with permit_guids for new amendments that may be for an existing permit';
-			UPDATE ETL_PERMIT SET permit_guid = (select permit_guid from permit WHERE ETL_PERMIT.permit_no=permit.permit_no and ETL_PERMIT.mine_guid = permit.mine_guid limit 1)
-			where permit_amendment_guid NOT IN (
-				SELECT permit_amendment_guid
-				FROM permit_amendment
-				);
+			UPDATE ETL_PERMIT SET permit_guid = (select permit_guid from permit
+				inner join mine_permit_xref mpx on mpx.permit_id = permit.permit_id
+				WHERE ETL_PERMIT.permit_no=permit.permit_no 
+				and ETL_PERMIT.mine_guid = mpx.mine_guid limit 1)
+            where permit_guid not in (select permit_guid from permit);--update permit_guids where permits weren't actually created
```

Comment:
Is the notice raised before this still relevant? Or still applicable? Are permit amendments now just normalized into being retrieved from the ETL_PERMIT table?